app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/ui/EditCoInsuredPresenter.kt

```diff
@@ -64,4 +69,5 @@ internal data class EditCoInsuredState(
   val isLoading: Boolean = false,
   val errorMessage: String? = null,
   val coInsured: ImmutableList<CoInsured> = persistentListOf(),
+  val member: Member? = null,
```

Comment:
This class atm can be of the possible 16 combinations of what is null and what is not null, (or false/true, or empty list/non-empty list).
Should a state like:
`loading = true, errorMessage = "error", coInsured = [blah], member = Member(...)` be possible?
How about the state:
`loading = false, errorMessage = null, coInsured = [], member = null`?

Both of those are possible here, but I am not sure if they should be possible in the first place?

Can use this https://github.com/HedvigInsurance/android/blob/5f1f762af6f5af12cc93946f7113bc4c9cee9a0a/app/feature/feature-login/src/main/kotlin/com/hedvig/android/feature/login/marketing/MarketingPresenter.kt#L60-L67 UiState as a good example. This could also be written as
```
internal data class MarketingUiState(
  val isLoading: Boolean,
  val market: Market?,
  val language: Language?
)
```
But then we make it possible for the state to be `isLoading = false, market = null, language = null`.
Or `isLoading = false, market = Market.SE, language = null`
Both of which are not useful for our UI to render, and are not possible to get to at all with the sealed interface solution instead.

Reply:
Makes sense, will change